kratos/geometries/geometry.h

```diff
@@ -921,6 +921,7 @@ class Geometry : public PointerVector<TPointType>
             auto norm2DXi = norm_2(DeltaXi);
 
             if(norm2DXi > 30 || norm2DXi < tol) {
```

Comment:
I would suggest to change the 30 with a constexpr

Reply:
Do I change it here or in a different PR?

Reply:
Just here would be better

Reply:
Also the rest of the constant numbers